introduction.md

```diff
@@ -5,3 +5,10 @@ Here I am, just a random developer chick, introducing myself to the dojo.
 Quite the cool thing, actually.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?